pkg/kubectl/resource/builder.go

```diff
@@ -696,3 +701,10 @@ func SplitResourceArgument(arg string) []string {
 	}
 	return out
 }
+
+// HasNames returns true if the provided args contain resource names
+func HasNames(args []string) bool {
+	args = normalizeMultipleResourcesArgs(args)
+	hasCombinedTypes, _ := hasCombinedTypeArgs(args)
```

Comment:
Does this function need to return the error, so that we can handle the error as early as possible?